backends/advanced/src/advanced_omi_backend/workers/conversation_jobs.py

```
@@ -303,6 +304,17 @@ async def open_conversation_job(
         conversation_id = conversation.conversation_id
         logger.info(f"✅ Created streaming conversation {conversation_id} for session {session_id}")
 
+    # Attach markers from Redis session (e.g., button events captured during streaming)
+    markers_json = await redis_client.hget(session_key, "markers")
```

Ruff (0.14.14)

[error] Undefined name `session_key`

(F821)